crates/ra_assists/src/assist_ctx.rs

```diff
@@ -136,6 +136,9 @@ impl<'a> AssistCtx<'a> {
     pub(crate) fn covering_node_for_range(&self, range: TextRange) -> SyntaxElement {
         find_covering_element(self.source_file.syntax(), range)
     }
+    pub(crate) fn lower_path(&self, path: ast::Path) -> Option<hir::Path> {
```

Comment:
this seems like a potential footgun, couldn't we have a method on `Semantics` that will infer the correct `FileId`?

Reply:
Agreed